js/block.js

```diff
@@ -6184,6 +6185,9 @@ function Block(protoblock, blocks, overrideName) {
         let that = this;
 
         let __selectionChanged = function() {
+            if(wheelValues[that._numberWheel.selectedNavItemIndex] === null) {
```

Comment:
If you use selectedValue, you don't need to do this. At line 6213, we already navigate to the selected value.

Reply:
so, should I do something more along the lines of:

```
 if(selectedValue != null) {
        labelElem.innerHTML =
            '<input id="numberLabel" style="position: absolute; -webkit-user-select: text;-moz-user-select: text;-ms-user-select: text;" class="number" type="number" value="' +
            selectedValue +
            '" />';
        }else {
            labelElem.innerHTML = '<input id="numberLabel" style="position: absolute; -webkit-user-select: text;-moz-user-select: text;-ms-user-select: text;" class="number" type="number" value="' +
            that.text.text +
            '" />';
        }
```

this works for the default value, but I'm not too sure if this is exactly right. Thanks for all the suggestions!

Reply:
I don't think you have to change anything in that code from the original. But you do need to modify the value of selectedValue to match one of the wheel values as per my earlier comment.